### PR TITLE
Couple matches

### PR DESCRIPTION
**Pull request by FazanaJ**
Making this asap since I made some pretty major renames

### CODE REVIEW

File: src/game.c [hunk 1]
```diff
@@ -140,11 +140,11 @@ Gfx *gDisplayLists[2];
 Gfx *gCurrDisplayList;
 UNUSED s32 D_801211FC;
 Mtx *gHudMatrices[2];
-Mtx *gCurrHudMat;
+Mtx *gGameCurrMatrix;
 VertexList *gHudVertices[2];
-VertexList *gCurrHudVerts;
+VertexList *gGamecurrVertexList;
```
Comment (Ryan-Myers) [inline src/game.c:145]:
```suggestion
VertexList *gGameCurrVertexList;
```
Will need to update all references as well.

File: src/game.c [hunk 2]
```diff
@@ -140,11 +140,11 @@ Gfx *gDisplayLists[2];
...
 TriangleList *gHudTriangles[2];
-TriangleList *gCurrHudTris;
+TriangleList *gGamecurrTriList;
```
Comment (Ryan-Myers) [inline src/game.c:147]:
```suggestion
TriangleList *gGameCurrTriList;
```

Same as above

File: src/game_ui.c [hunk 3]
```diff
@@ -169,42 +173,42 @@ s32 D_800E283C[5] = {
...
+s8 D_80126CD0;
+s8 D_80126CD1;
+s8 D_80126CD2;
+s8 D_80126CD3;
+s8 D_80126CD4;
+s8 D_80126CD5;
 s32 D_80126CD8;
 unk80126CDC *D_80126CDC;
-s32 D_80126CE0;
+unk80126CDC *D_80126CE0[1];
```
Comment (Ryan-Myers) [inline src/game_ui.c:184]: **`D_80126CE0` to `D_80126CEC` are all unused. Should this be a bigger array?**

**Review by Ryan-Myers — APPROVED** [review]
LGTM, just a couple minor notes.

File: src/game_ui.c (hunk 3 above)
Comment (FazanaJ) [reply src/game_ui.c]: I think there may be more than 3 indexes, but I'll set it to that, for the time being

**Pull request merged**